docs/conf.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+from pathlib import Path
+
+# -- Sphinx Options -----------------------------------------------------------
+
+# If your project needs a minimal Sphinx version, state it here.
+needs_sphinx = "3.0"
+
+extensions = [
+    "sphinx.ext.autodoc",
+    "sphinx.ext.extlinks",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.todo",
+    "sphinxarg.ext",  # Automatic argparse command line argument documentation
+]
+
+autodoc_default_options = {
+    "members": True,
+}
+
+# The suffix(es) of source filenames.
+source_suffix = {
+    ".rst": "restructuredtext",
+    # '.txt': 'markdown',
+    # '.md': 'markdown',
+}
+
+master_doc = "index"
+
+project = u"Introduction to TDD and CI With VUnit"
+copyright = u"2014-2021, Lars Asplund"
+author = u"LarsAsplund, kraigher and contributors"
```

Comment:
I'm not sure kraigher is an author in this repo. Maybe just Lars Asplund and contributors.